src/watchmaker/static/config.yaml

```diff
@@ -3,7 +3,12 @@ all:
       admin_groups: None
       admin_users: None
       computer_name: None
-      environment: False
+      environment:
+      valid_environments:
```

Comment:
We can remove this from the default config. These values are implementation-specific, and we try not to impose implementation-specific values on users via the default config...

Reply:
I'm still seeing this change in the PR... This file should not be changed in this PR.

Reply:
Fixed.